src/_blog/metrics-in-go-libp2p.md

```diff
@@ -0,0 +1,162 @@
+1. Dial ranking delay. This metric tracks the latency in connection establishment introduced by the dial prioritisation logic. 
+2. Dials per connection. This metric counts the number of addresses dialed before a connection was established with the peer. 
+
+Dials per connection measured the benefit of introducing smart dialing mechanism, and dial ranking delay provided us with the assurance that the vast majority of dials had no adverse impact on latency. 
+
+<div class="container" style="display:flex; column-gap:10px; justify-content: center; align-items: center;">
+    <figure>
+        <img src="../assets/metrics-in-go-libp2p-smart-dialing.png" width="750">
+        <figcaption style="font-size:x-small;">
+                Smart dialing metrics
+        </figcaption>
+    </figure>
+</div>
+
+
+## Resources
+
+Check out our grafana dashboards: [https://github.com/libp2p/go-libp2p/tree/master/dashboards](https://github.com/libp2p/go-libp2p/tree/master/dashboards)
+
+To create custom dashboards, the [prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/) and [grafana docs](https://grafana.com/docs/grafana/latest/panels-visualizations/) are great resources.
+
+
+## Get Involved
+
+- If you’d like to get involved and contribute to libp2p, you can reach out to us using these means: [https://libp2p.io/#community](https://libp2p.io/#community)
+- If you’re a self-starter and want to start pushing code immediately, feel free to ping the maintainers in any of these help wanted/good first issues: [go-libp2p](https://github.com/libp2p/go-libp2p/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), [js-libp2p](https://github.com/libp2p/js-libp2p/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), and [rust-libp2p](https://github.com/libp2p/rust-libp2p/issues?q=is%3Aopen+is%3Aissue+label%3Agetting-started).
+- If you want to work in and around libp2p full-time, there are various teams hiring including the implementation teams.  See [https://jobs.protocol.ai/jobs?q=libp2p](https://jobs.protocol.ai/jobs?q=libp2p) for opportunities across the [Protocol Labs Network](https://plnetwork.io/).
+
+To learn more about libp2p generally, checkout:
+
+- The [libp2p documentation portal](https://docs.libp2p.io/)
+- The [libp2p connectivity website](https://connectivity.libp2p.io/)
+- The [libp2p curriculum put together by the Protocol Labs Launchpad program](https://curriculum.pl-launchpad.io/curriculum/libp2p/introduction/)
+
+You can reach out to us and stay tuned for our next event announcement by joining our [various communication channels](https://libp2p.io/#community), joining the [discussion forum](https://discuss.libp2p.io/), following us on [Twitter](https://twitter.com/libp2p), or saying hi in the #libp2p-implementers channel in the [Filecoin public Slack](http://filecoin.io/slack).
```

Comment:
Do we need this? This sounds like something that should be in the footer of the blog itself, not part of the blog post.

Reply:
@p-shahi wdyt? 
I added this because most other posts have a `Resources and how you can help contribute 💪` section. I don't mind removing this from here but I think this should be available somewhere.

Reply:
I prefer to keep it here as a call to action for now.
Happy to move the generic info to a footer in the future. Only worry about that is how to present it so the footer is not too verbose